.github/workflows/run-tpch.yaml

```diff
@@ -0,0 +1,164 @@
+name: Run tpch benchmarks
+
+on:
+  workflow_dispatch:
+    inputs:
+      wheel:
+        type: string
+        description: The wheel artifact to use
+        required: false
+        default: getdaft-0.3.0.dev0-cp38-abi3-manylinux_2_31_x86_64.whl
+      skip_questions:
+        type: string
+        description: The TPC-H questions to skip
+        required: false
+        default: ""
+      scale_factor:
+        type: choice
+        options:
+        - '2'
+        - '10'
+        - '100'
+        - '1000'
+        description: Which scale factor to use
+        required: false
+        default: '2'
+      partition_size:
+        type: choice
+        options:
+        - '2'
+        - '32'
+        - '32'
+        - '100'
+        - '300'
+        - '320'
+        - '512'
+        description: Which partition size to use
+        required: false
+        default: '2'
+      python_version:
+        type: string
+        description: The version of python to use
+        required: false
+        default: "3.9"
+  workflow_call:
+    inputs:
+      wheel:
+        type: string
+        description: The wheel artifact to use
+        required: false
+        default: getdaft-0.3.0.dev0-cp38-abi3-manylinux_2_31_x86_64.whl
```

Comment:
What's this version of Daft and why is this not required?

Reply:
This is the name of the wheel which is inside of the directory in the `github-actions-artifacts-bucket` bucket. The name of the outputted wheel is determined via `build` job which then injects the name of the wheel to this step. If you want to run this step without having run the previous, you'll have to specify the name.

It's possible that we have an S3 List job to grab the name instead...